### PR TITLE
Editorial mdlink and vale updates for HowTo/Configure directory

### PR DESCRIPTION
**Pull request by grantnoble**
The usual culprits for mdlint and vale edits. Plus changes to some headings, and some wording changes and misc. punctuation

### CODE REVIEW

File: docs/HowTo/Configure/FreeGas.md [hunk 1]
```diff
@@ -1,54 +1,60 @@
...
-The transaction cost is paid to the miner (or validator in PoA networks) that includes the transaction in a block.  
+In public networks, the account submitting the transaction pays the transaction cost, in Ether.
+The miner (or validator in PoA networks) that includes the transaction in a block receives
+transaction cost.
 
-In many private networks, the validators are run by the network participants and do not require gas as an 
-incentive to participate.  Generally, networks that do not require gas as an incentive, configure the gas price to be 0 (that is, make gas free). 
-Some private networks may allocate Ether and use a non-zero gas price to limit resource use.  
+In many private networks, network participants run the validators and do not require gas as an
+incentive. Generally, networks that do not require gas as an incentive, configure the gas price to
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/FreeGas.md]: Feels like a word is missing in this sentence? 

Generally, in networks ....

File: docs/HowTo/Configure/Genesis-File.md [hunk 2]
```diff
@@ -1,20 +1,23 @@
-description: Configuring network using genesis file  
+description: Configuring a network using the genesis file
 <!--- END of page meta data -->
 
-# Creating Hyperledger Besu Genesis File 
+# Creating the Hyperledger Besu genesis file
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Genesis-File.md]: Did you search the doc repo for links to this heading as discussed on Friday?

File: docs/HowTo/Configure/Genesis-File.md [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
 
-The genesis file defines the first block in the chain and the first block defines which chain you want to join.
+The genesis file defines the first block in the chain and the first block defines which chain you
+want to join.
 
-For Ethereum MainNet and public testnets (for example, Rinkeby), the genesis configuration is defined in 
-Besu and used when a public network is specified using the [`--network`](../../Reference/CLI/CLI-Syntax.md#network)
-command line option. 
+For Ethereum mainnet and public testnets (for example, Rinkeby) the genesis configuration
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Genesis-File.md]: MainNet is in our style guide

File: docs/HowTo/Configure/Using-Configuration-File.md [hunk 4]
```diff
@@ -1,32 +1,37 @@
-description: Using Hyperledger Besu Configuration File
+description: Using the Hyperledger Besu configuration file
 <!--- END of page meta data -->
 
-# Using Hyperledger Besu Configuration File
+# Using the Hyperledger Besu configuration file
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Using-Configuration-File.md]: Same comment as above - have you searched for and updated any links to this heading?

File: docs/HowTo/Configure/FreeGas.md (hunk 1 above)
Comment (grantnoble) [reply docs/HowTo/Configure/FreeGas.md]: I agree the sentence is a bit poorly worded. I changed to:
"Networks that do not require gas as an incentive usually configure the gas price to
be zero (that is, make the gas free)."

File: docs/HowTo/Configure/Genesis-File.md (hunk 2 above)
Comment (grantnoble) [reply docs/HowTo/Configure/Genesis-File.md]: Yes, every time I make a heading change I search for links to that heading throughout the repo, and will update any I find and include them in the PR.

File: docs/HowTo/Configure/Genesis-File.md (hunk 3 above)
Comment (grantnoble) [reply docs/HowTo/Configure/Genesis-File.md]: Ah, OK. I've seen other instances of "mainnet" which I will change if I see them again.

File: docs/HowTo/Configure/Using-Configuration-File.md (hunk 4 above)
Comment (grantnoble) [reply docs/HowTo/Configure/Using-Configuration-File.md]: Yep.

**Pull request merged**